app/views/ProjectEdit/ProjectDetailsForm/index.tsx

```diff
@@ -202,6 +202,12 @@ function ProjectDetailsForm(props: Props) {
         setValue,
     } = useForm(schema, initialValue);
 
+    const {
+        user,
+    } = useContext(UserContext);
+
+    const accessPrivateProject = user?.accessibleFeatures?.some((f) => f.key === 'PRIVATE_PROJECT');
```

Comment:
```suggestion
    const accessPrivateProject = !!user?.accessibleFeatures?.find((f) => f.key === 'PRIVATE_PROJECT');
```

Reply:
Is the logic already implemented for accessPrivateProject=true condition?

Reply:
yes